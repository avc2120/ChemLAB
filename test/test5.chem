/* Test 5: If Conditional, Boolean */

function main() {
	int X;
	int Y;
	X = 17;
	Y = 42;
	if (X < Y) {
		print X + " is less than " + Y;
	} else {
		print "Test Failed";
	}
}