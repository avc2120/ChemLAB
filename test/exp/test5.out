17 is less than 42